client/src/app/gearset-list/gearset-list.component.ts

```diff
@@ -59,10 +50,39 @@ export class GearsetListComponent implements OnInit {
 		mp: -1491.90,
 		dps: 13621.75,
 	}];
+	editCache: { [key: string]: { edit: boolean; data: GearSet } } = {};
```

Comment:
this is good for now but we'll want to use a backend database soon

Reply:
Yeah, saving the gearsets permanently would be A+